README.md

```diff
@@ -10,67 +10,141 @@ Django Major Event Log App [![Build Status](https://travis-ci.org/unt-libraries/
-
-* Event ID
-* Title
-* Details
-* Outcome
-* Outcome Details
-* Event Date
-* Event Creation Date
-* Event Last Modified Date
-* Contact Name
-* Contact Email
-
-The **Index** page displays all of the created events, in reverse chronological
-order (most recent Event Date first). The title, event date, ID, and
-outcome are all displayed on this page. The title of each event is
-clickable and takes you to the details page for that specific event.
-
-The **Event Details** page displays all the information about a specific event.
-On top of displaying all the information in a human-readable form, this page
-provides links to the premis xml representation of the event, as well as an
-atom item for the event, which contains all the premis xml wrapped in the atom
-item.
-
-The **Atom Feed** page, linked to in the navigation bar of each html page,
-provides an atom feed for the ten most recent events. Again, these are
-sorted by the event date, not the creation or modified dates.
-
-Finally, there is also a small `About` page which contains a description of the
-application.
+
+Requirements
+--------------------------
+
+- Django == 1.8
+- Python >= 2.7
+
 
 Installation
 --------------------------
 
-1.  Download Source
+1. Install with the following command (in the root of the app directory)
+
+        $ pip install major-event-log
+
+2. Add `major_event_log` to your `INSTALLED_APPS`. Be sure to add `django.contrib.admin` (and its dependencies) and `django.contrib.humanize` if they are not already present. Your `INSTALLED_APPS` should look like the following (at the minimum):
+
+        INSTALLED_APPS = (
+          'django.contrib.admin',
+          'django.contrib.auth',
+          'django.contrib.contenttypes',
+          'django.contrib.sessions',
+          'django.contrib.messages',
+          'django.contrib.staticfiles',
+          'major_event_log',
+        )
+
+3. Include the URLs
+
+        urlpatterns = [
+          url(r'^admin/', include(admin.site.urls)),
+          url(r'^major-event-log/', include('major_event_log.urls',
+            namespace="major-event-log"))
+        ]
+
+4. Migrate/sync the database
+
+        $ python manage.py migrate major-event-log
 
-    To get the application, download the source code from the Github repository
-    [here](https://github.com/unt-libraries/django-major-event-log).
+5. Configure static files
+
+        STATIC_URL = '/static/'
 
-2.  Install
 
-    Install with `pip install https://github.com/unt-libraries/django-major-event-log`
-    
 License
 -------------------------
 
 See LICENSE
 
+
 Contributors
 -------------------------
 
-Django-major-event-log was developed at the UNT Libraries.
+django-major-event-log was developed at the UNT Libraries.
 
 Contributors:
 
 * [Gio Gottardi](https://github.com/somexpert)
+* [Damon Kelley](https://github.com/damonkelley)
+
+
+Developing
+-------------------------
+
+#### Requirements ####
```

Comment:
Can we remove the Requirements section and instead just add a section to the next section that instructs him/her to run 

```sh
$ pip install -r requirements.txt
```
?